src/app/company-dashboard/company-settings/company-settings.component.html

```diff
@@ -17,7 +17,7 @@ <h5>
                 <app-location-form [inCompany]="true"></app-location-form>
                 <hr>
                 <label class="bold">Tracked Utilities</label>
-                <app-units-form [inCompany]="true"></app-units-form>
+                <app-units-form></app-units-form>
```

Comment:
We will want to remove this whole section in company-settings related to Tracked Utilities

Reply:
good catch